test/runtime-integration/node.cjs

```diff
@@ -4,8 +4,12 @@ const fs = require("fs");
 
 const [major, minor] = process.versions.node.split(".").map(n => +n);
 
-if (major > 12 || (major === 12 && minor >= 17)) {
-  test("ESM", "--experimental-modules ./src/main-esm.mjs", "expected-esm.txt");
+if (
+  major > 13 ||
+  (major === 12 && minor >= 17) ||
+  (major === 13 && minor >= 2)
```

Comment:
ESM is not enabled by default on 13.0 and 13.1 (thanks @JLHwung for pointing it out)